.devcontainer/Dockerfile

```diff
@@ -310,14 +312,85 @@ RUN git clone https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter.git
     make -j"$(nproc)" && \
     make install && \
     cd / && \
-    rm -rf freediameter
+    rm -rf freediameter \
+    rm -rf /tmp/*
 
 ##### Go language server support for vscode
 RUN go get -v golang.org/x/tools/gopls
 
 #### Update shared library configuration
 RUN ldconfig -v
 
+
+##### Install Python requirements
+
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+
+### install egg-links
+COPY / ${MAGMA_ROOT}
+# need to include entire magma repo to replicate make file command that generates proto files, but there is a helpful .dockignore
+
+WORKDIR ${MAGMA_ROOT}/lte/gateway/python/
+RUN pip3 install --quiet --upgrade --no-build-isolation --no-cache-dir --verbose --editable .[dev]
+
+WORKDIR ${MAGMA_ROOT}/orc8r/gateway/python/
+RUN pip3 install --quiet --upgrade --no-build-isolation --no-cache-dir --verbose --editable .[dev]
+
+# chown -R ${USER_UID}:${USER_GID} ${PIPX_HOME}  # muss ich permissions anpassen fuer non-root user?
+
+#### protos
+WORKDIR $MAGMA_ROOT
+
+ARG PYTHON_LIB=/usr/local/lib/python3.8
+
+RUN mkdir $PYTHON_LIB/gen \
+    && for PROTO_SRC in orc8r lte feg; \
+    do \
+    python protos/gen_protos.py ${PROTO_SRC}/protos ${MAGMA_ROOT},orc8r/protos/prometheus ${MAGMA_ROOT} $PYTHON_LIB/gen \
+    && python protos/gen_prometheus_proto.py ${MAGMA_ROOT} $PYTHON_LIB/gen; \
+    done \
+    && echo "$PYTHON_LIB/gen" > $PYTHON_LIB/dist-packages/magma_gen.pth \
+    && find $PYTHON_LIB/gen -type d | tail -n +2 | sed '/__pycache__/d' | xargs -I % touch "%/__init__.py"
+    # fuegt allen directories, bis auf gen, init dateien hinzu
+
+### swagger
+ENV SWAGGER_CODEGEN_JAR=/var/tmp/codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar
+ARG MVN_VERSION=3.5.4
+ARG CODEGEN_VERSION=v2.2.3
+
+WORKDIR /var/tmp/
+
+RUN wget http://mirrors.ibiblio.org/apache/maven/maven-3/$MVN_VERSION/binaries/apache-maven-$MVN_VERSION-bin.tar.gz \
+    && tar --extract --verbose --file apache-maven-$MVN_VERSION-bin.tar.gz
+
+RUN git clone --branch $CODEGEN_VERSION https://github.com/swagger-api/swagger-codegen /var/tmp/codegen
+
+WORKDIR /var/tmp/codegen
+RUN ../apache-maven-$MVN_VERSION/bin/mvn clean package -DskipTests
+
+RUN for SWAGGER_SRC in lte orc8r; \
+     do \
+     mkdir -p $PYTHON_LIB/gen/$SWAGGER_SRC/swagger/specs \
+     && touch $PYTHON_LIB/gen/$SWAGGER_SRC/swagger/__init__.py \
+     && touch $PYTHON_LIB/gen/$SWAGGER_SRC/swagger/specs/__init__.py \
```

Comment:
Have you tried this without those `__init__.py`s? We might not need those with Python 3.

Reply:
Also, I think it might be a good idea to migrate this to Bazel now.

Reply:
> Have you tried this without those `__init__.py`s? We might not need those with Python 3.

For some reason leaving out the __init__.py files does cause problems for some proto and swagger files but not for others. Meaning they are not resolved properly anymore.